src/Sylius/Component/Order/Context/ResettingCartContextInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Order\Context;
+
+interface ResettingCartContextInterface
```

Comment:
As I wrote in other comment – in Sylius `2.0` we're going to add the `reset` method directly on the `CartContextInterface`, so let's add a deprecation note here. Just in case :P.

```suggestion
/** @deprecated since 1.13 and will be removed in Sylius 2.0. */
interface ResettingCartContextInterface
```

Reply:
If you keep the `ResettingCartContextInterface` in 2.0 and make `CartContextInterface` extend it, then there is no need for the deprecation.

Reply:
I know we can keep `ResettingCartContextInterface`, but we had many discussions how to handle it. AFAIR our ideal solution was to add this method to the `CartContextInterface`, but we couldn't due to BC policy.